qiskit_experiments/library/randomized_benchmarking/clifford_utils.py

```diff
@@ -299,6 +299,22 @@ def _unpack_num(num, sig):
         return res
 
 
+def compute_target_bitstring(circuit: QuantumCircuit) -> str:
+    """For a Clifford circuit C, compute C|0>.
+    Args:
+        circuit: A Clifford QuantumCircuit.
+    Returns:
+        Target bitstring.
+    """
+    # convert circuit to Boolean phase vector of stabilizer table
+    phase_vector = Clifford(circuit).table.phase
+    n = circuit.num_qubits
+
+    # target string has a 1 for each True in the stabilizer half of the phase vector
+    target = "".join(["1" if phase else "0" for phase in phase_vector[n:][::-1]])
```

Comment:
`stab_phase` method in Clifford is useful here. You can replace `phase_vector[n:]` with `Clifford(circuit).stab_phase`.

Reply:
Thanks, done in 09fe450.